azurelinuxagent/daemon/main.py

```diff
@@ -69,7 +69,6 @@ def run(self, child_args=None):
         self.initialize_environment()
 
         CGroupConfigurator.get_instance().create_agent_cgroups(track_cgroups=False)
-        CGroupConfigurator.get_instance().cleanup_old_cgroups()
```

Comment:
Why not have the daemon clean up?

Reply:
It is not needed. Or did you have an scenario in mind when it was added?